ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/ConnectIntegrationTest.java

```diff
@@ -326,11 +361,27 @@ public void shouldWriteTimestampsToConnect() throws UnsupportedEncodingException
         stringContainsInOrder(sinkOutputParts), TIMEOUT_NS, TimeUnit.NANOSECONDS);
   }
 
+  @Test
+  public void shouldListConnectorPlugins() {
+    // Given:
+    create("mock-connector", ImmutableMap.of(
```

Comment:
Is this needed? We shouldn't need to create a connector in order to list plugins.